src/procedures/utility/build_matrix.c

```
@@ -0,0 +1,166 @@
+				_A, _A, GrB_DESC_T1);
+		ASSERT(info == GrB_SUCCESS);
+	}
+
+	if(compact) {
+		// determine the number of nodes in the graph
+		// this includes deleted nodes
+		size_t n = Graph_UncompactedNodeCount(g);
+
+		// get rid of extra unused rows and columns
+		GrB_Info info = GrB_Matrix_resize(_A, n, n);
+		ASSERT(info == GrB_SUCCESS);
```

Cppcheck (2.10-2)

[error] Null pointer dereference

(nullPointer)